<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="1" ignored="0" total="4" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="3322" started-at="2022-07-01T10:27:09Z" finished-at="2022-07-01T10:27:12Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="3322" started-at="2022-07-01T10:27:09Z" finished-at="2022-07-01T10:27:12Z">
      <class name="com.tests.Logintest">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.tests.Logintest@5a8e6209]" name="setup" is-config="true" duration-ms="10447" started-at="2022-07-01T10:26:58Z" finished-at="2022-07-01T10:27:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="failtest()[pri:0, instance:com.tests.Logintest@5a8e6209]" name="failtest" duration-ms="22" started-at="2022-07-01T10:27:09Z" finished-at="2022-07-01T10:27:09Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.tests.Logintest.failtest(Logintest.java:42)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- failtest -->
        <test-method status="PASS" signature="passtest()[pri:0, instance:com.tests.Logintest@5a8e6209]" name="passtest" duration-ms="2483" started-at="2022-07-01T10:27:09Z" finished-at="2022-07-01T10:27:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- passtest -->
        <test-method status="SKIP" signature="skiptest()[pri:0, instance:com.tests.Logintest@5a8e6209]" name="skiptest" duration-ms="3" started-at="2022-07-01T10:27:12Z" finished-at="2022-07-01T10:27:12Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[We are skipping this]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: We are skipping this
at com.tests.Logintest.skiptest(Logintest.java:48)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- skiptest -->
        <test-method status="PASS" signature="dashboardtest()[pri:0, instance:com.tests.Logintest@5a8e6209]" name="dashboardtest" duration-ms="254" started-at="2022-07-01T10:27:12Z" depends-on-methods="com.tests.Logintest.passtest" finished-at="2022-07-01T10:27:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- dashboardtest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.tests.Logintest@5a8e6209]" name="tearDown" is-config="true" duration-ms="1444" started-at="2022-07-01T10:27:12Z" finished-at="2022-07-01T10:27:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.tests.Logintest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
